save.rb

```diff
@@ -41,6 +40,17 @@ def save_game(game)
     File.write('./data/game.json', JSON.pretty_generate(game_arr))
   end
 
+  def dave_author(author)
```

Comment:
I suppose that you meant to name this method `save_author` instead of `dave_method` if that is the case then I would like to bring your attention to this possible typo